tests/unit/preprocessor/_regrid/test_regrid.py

```diff
@@ -265,50 +267,126 @@ def make_test_cube(shape):
     return cube
 
 
-def test_rechunk_on_increased_grid():
-    """Test that an increase in grid size rechunks."""
-    with dask.config.set({'array.chunk-size': '128 M'}):
+def make_test_cube_unstructured(shape):
```

Comment:
I think this defines an irregular cube (using [our definitions](https://esmvaltool--2457.org.readthedocs.build/projects/ESMValCore/en/2457/recipe/preprocessor.html#grid-types)).

Reply:
I mixed those two up

Reply:
Fixed now